PADx_scraper.py

```diff
@@ -75,12 +114,12 @@ def main():
     file = input("Destination file: ")
 
     # Extract and format HTML from URL
-    rem_src = requests.get(url)
-    rem_plain_txt = rem_src.text
-    rem_soup = BeautifulSoup(rem_plain_txt, features="html.parser")
-    print("Scrapping...")
-
-    scrape(rem_soup, file)
+    
+    print("Scraping...")
+    start = time.perf_counter()
+    scrape(url, file)
+    end = time.perf_counter()
+    print(f"Scraping completed in {end - start} seconds.")
 
 if __name__ == "__main__":
     main()
```

Comment:
Mind adding a newline at the end? my bad I didn't add one earlier.

Reply:
Took 2 seconds ;)